### PR TITLE
Update package.json

### LINKED ISSUE #110: Remove "auto" related dependencies
We tried adding "auto" to auto-generate releases, but it didn't work as expected. We reverted the changes done to circleCI config file as part of https://github.com/intuit/user-data-for-fraud-prevention/pull/74 for this. However, the changes done in package.json were not reverted. Please remove all "auto" related code in package.json as we are not using this.

### PR DESCRIPTION
**Pull request by vvijayalakshmi21**
Fix #110 and https://github.com/intuit/user-data-for-fraud-prevention/issues/118

# Title
Removed all ref to auto

## Description of what's changing
package.json 

## What else might be impacted?

...

## Which issue does this PR relate to?

...

## Checklist

[ ] Tests are written and maintain or improve code coverage
[ ] I've tested this in my application using `yarn link` (if applicable)
[ ] You consent to and are confident this change can be released with no regression

### CODE REVIEW

**Comment by skodamarthi** [comment]
@vvijayalakshmi21 Looks good! Can you please run "yarn" and also commit yarn.lock file ?

**Comment by vvijayalakshmi21** [comment]
image

I see this
Please can you help>?

File: README.md [hunk 1]
```diff
@@ -23,20 +23,8 @@ Each top level folder in `src/js` has its own README with more specific informat
 ## How to use
 Usage instructions can be found [here](./USAGE.md)
 
-## Demo
-### Steps for testing:
+## How to test
```
Comment (skodamarthi) [inline README.md:26]: Under How to test section, please add "Instructions for testing the changes can be found [here](./Demo.md)"

**Review by skodamarthi — COMMENTED** [review]
@vvijayalakshmi21 Please check the review comments.

**Comment by vvijayalakshmi21** [comment]
Sorry for committing two issues in one PR.
Please check now

**Comment by skodamarthi** [comment]
> image
> 
> I see this
> Please can you help>?

@vvijayalakshmi21 For commit messages, you need to follow the format that is suggested in the error message by commitlint: Eg: chore: remove auto related dependencies.

You can read more about conventional commit message formats here: https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-conventional

**Comment by vvijayalakshmi21** [comment]
Fix for #118 also included

**Review by skodamarthi — APPROVED** [review]

**Comment by skodamarthi** [comment]
@vvijayalakshmi21 Changes look good! We will merge now. Thank you for contributing!

**Comment by skodamarthi** [comment]
@all-contributors please add @vvijayalakshmi21 for docs and maintenance.

**Pull request merged**